src/Ganeti/Confd/Client.hs

```diff
@@ -40,9 +40,11 @@ module Ganeti.Confd.Client
 import Control.Concurrent
 import Control.Exception (bracket)
 import Control.Monad
+import Data.ByteString.Char8 (pack, unpack)
```

Comment:
Uh, most of the code here is dealing network I/O. In this context, my first thought is that `Data.ByteString.Char8` is not appropriate, and that instead plain `Data.ByteString` should be used. But maybe I'm missing some thing?